### ISSUE TITLE
Explain dual-licensing page position better

### ISSUE THREAD

**Issue by Beanow** [sfosc/sfosc#81]
The current dual-licensing page https://sfosc.org/business-models/dual-licensing/
take a fairly resolute position against it's use for SFOSC.

> #### Does this model help create a Sustainable Free and Open Source Community?
> No. It trades the fundamental liberties in the core commitment in exchange for revenue.

A lot of debate about this position already happened in #42 and #63.
From which _I think_ the takeaways are:
- SFOSC is opinionated about this and that's OK.
- Given how much debate went into it, we can dedicate a few more words on why this position is taken.

So this issue is about **adding more detail** explaining more of the considerations why dual-licensing is not a good fit for SFOSC.

If you're willing to help with a pull request, please keep the above in mind.

To discuss the standpoint itself, please open a new issue first to discuss it with the community before creating a pull request :smile:

**Comment by pradyunsg** [comment]
The appropriate link is now: https://sfosc.org/docs/business-models/dual-licensing/

**Comment by Beanow** [comment]
You're right, thank you @pradyunsg :smile:

**Comment by pradyunsg** [comment]
> #### Does this model help create a Sustainable Free and Open Source Community?
> 
> No. It trades the fundamental liberties in the core commitment in exchange for revenue.

Is the thing needed here, a more elaborate and detailed answer to the final question in the relevant page?

**Comment by Beanow** [comment]
Yes. I feel like the referenced issues have touched on a couple of angles to analyse this standpoint and it would be good to include them on the page. I don't have a summary of those at hand currently.
